gflownet/utils/crystals/constants.py

```diff
@@ -612,241 +669,243 @@
 # Point group of each space group, from pymatgen
 # See: ./scripts/pymatgen/pg_of_all_sg.py
 # Each item in the dictionary contains a list of:
-# - point group as in pymatgen
-# - crystal class index of the point group
-# - crystal system index
-# - point symmetry index
+# - [0] point group as in pymatgen
+# - [1] crystal class index of the point group
+# - [2] crystal system index
+# - [3] lattice system index
+# - [4] crystal-lattice system index
+# - [5] point symmetry index
 SPACE_GROUPS = {
```

Comment:
Is this (and other constants extracted from .yaml files now) still in use? Correct me if I am wrong, but my impression was that this was supposed to be removed in favor of yamls?